Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "gwasm-runner"
-version = "0.4.0"
+version = "0.4.2"
```

Why to versions at once? :)
```suggestion
version = "0.4.1"
```